passiveData/src/commonMain/kotlin/org/sagebionetworks/assessmentmodel/passivedata/asyncaction/AsyncActionConfiguration.kt

```diff
@@ -0,0 +1,31 @@
+package org.sagebionetworks.assessmentmodel.passivedata.asyncaction
+
+/**
+ *  `AsyncActionConfiguration` defines general configuration for an asynchronous background action
+ * that should be run in the background. Depending upon the parameters and how the action is set
+ * up, this could be something that is run continuously or else is paused or reset based on a
+ * timeout interval.
+ *
+ * The configuration is intended to be a serializable object and does not call services, record
+ * data, or anything else.
+ *
+ * - seealso: `AsyncActionController`.
+ *
+ */
+
+
+interface AsyncActionConfiguration {
```

Comment:
Shouldn't this be `org.sagebionetworks.assessmentmodel.AsyncActionConfiguration` since this repo depends on that one?